about.html

```diff
@@ -241,10 +241,10 @@
                 <a class="button3 clicky">Decisions</a>
                 <a class="button4 clicky">Steering Council</a>
                 <div class="items">
-                    <div class="div1 content-gov abs">Project Jupyter is part of the non-profit NumFOCUS Foundation, which provides fiscal sponsorship, governance and a legal umbrella for the project.</div>
-                    <div class="div2 content-gov abs">We use a shared copyright model that enables all contributors to maintain the copyright on their contributions. All code is licensed under the terms of the revised BSD license.</div>
-                    <div class="div3 content-gov abs">Technical decisions are made through open community discussion and consensus, with a Steering Council and BDFL (Fernando Perez) stepping in when consensus cannot be reached.</div>
-                    <div class="div4 content-gov abs">The overall leadership, strategy and direction for the project is provided by the Jupyter Steering Council, a group of the most active project contributors that are nominated for leadership.</div>
+                    <div class="div1 content-gov">Project Jupyter is part of the non-profit NumFOCUS Foundation, which provides fiscal sponsorship, governance and a legal umbrella for the project.</div>
+                    <div class="div2 content-gov" style='display:none'>We use a shared copyright model that enables all contributors to maintain the copyright on their contributions. All code is licensed under the terms of the revised BSD license.</div>
+                    <div class="div3 content-gov" style='display:none'>Technical decisions are made through open community discussion and consensus, with a Steering Council and BDFL (Fernando Perez) stepping in when consensus cannot be reached.</div>
+                    <div class="div4 content-gov" style='display:none'>The overall leadership, strategy and direction for the project is provided by the Jupyter Steering Council, a group of the most active project contributors that are nominated for leadership.</div>
```

Comment:
The `style='display:none'` avoid the flash of the content on page loading. Absolute positionning is always a bad idea, you always get into edge cases : Current Jupyter website  at some specific width. 
![screen shot 2015-10-21 at 11 31 54](https://cloud.githubusercontent.com/assets/335567/10646849/5eadade8-77ea-11e5-87d8-3dbe822e64fc.png)

the abs-positionning, forced you to set the height of element in css which you will get wrong.